src/openrct2/platform/Platform.Posix.cpp

```diff
@@ -68,6 +68,7 @@ namespace Platform
         if (uname(&buffer) < 0)
         {
             log_error("Couldn't get OS");
+            Console::WriteLine(std::strerror(errno));
```

Comment:
You should include Console.h(pp?) if you use its functions.

Reply:
I included it in line 14 already, I believe.